exercises/space-age/tests/space-age.rs

```diff
@@ -0,0 +1,81 @@
+// Students,
+//
+// These tests use the `num` crate to test that floating-point numbers are within an expected
+// range.
+//
+// The first time you run `cargo test`, the Num crate will automatically be downloaded
+// and installed. More information on crates can be found at
+// https://doc.rust-lang.org/book/guessing-game.html#generating-a-secret-number
+//
+extern crate space_age;
+extern crate num;
+
+use std::fmt::Display;
+use space_age::*;
+use num::{Float, NumCast};
+
+fn assert_in_delta<T: Float + Display>(expected: T, actual: T) {
```

Comment:
Hmm. I swear I was having problems with comparisons in some cases, but now I can't re-create them. I'll drop the crate and see if I run into any problems.